example/SimpleApp/platformio.ini

```diff
@@ -30,5 +39,63 @@ upload_protocol = esptool
 custom_factory_name = factory ;name of the binary file
 custom_factory_url = 
 
+[env:adafruit_feather_esp32_v2-app]
+board = adafruit_feather_esp32_v2
+board_build.partitions = ../../partition/8MB.csv
+board_build.filesystem = littlefs
+
+[env:adafruit_matrixportal_s3-app]
+board = adafruit_matrixportal_esp32s3
+board_build.partitions = partition/8MB.csv

```

Comment:
IMHO in the original partition table, there were additional partitions, which are missing now!?